data/mods/Magiclysm/Spells/kelvinist.json

```diff
@@ -5,7 +5,7 @@
     "name": "Point Flare",
     "description": "Causes an intense heat at the location, damaging the target.",
     "valid_targets": [ "hostile" ],
-    "flags": [ "CONCENTRATE", "SOMATIC", "LOUD" ],
+    "flags": [ "CONCENTRATE", "SOMATIC", "LOUD", "IGNITE_FLAMMABLE" ],
```

Comment:
this spell should not ignite things.

Reply:
why not

Reply:
because i don't want it to - it's not a "fire" spell, it's a "heat" spell.

Reply:
understand, overheatting wont ignite things, it will directly reduce things into ashes, you mean that right? but maybe to make it more sensible, I can further let spells able to be cast on terrain and destroy them, and also make a flag that destroy items like vehicle wheel going over?